sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -53,48 +58,78 @@ dependencies {
   compile(library.java.google_cloud_core_grpc) {
     exclude group: 'io.grpc', module: 'grpc-core' // Use Beam's version
   }
+  permitUnusedDeclared library.java.google_cloud_core_grpc // BEAM-11761
   compile library.java.google_cloud_datastore_v1_proto_client
   compile library.java.google_cloud_pubsublite
   // GCP PubSub client is used in TestPubSub
   compile library.java.google_cloud_pubsub
   compile library.java.google_cloud_spanner
+  compile library.java.google_code_gson
   compile library.java.google_http_client
   compile library.java.google_http_client_jackson2
   compile library.java.grpc_alts
+  permitUnusedDeclared library.java.grpc_alts // BEAM-11761
   compile library.java.grpc_api
   compile library.java.grpc_auth
   compile library.java.grpc_core
+  permitUnusedDeclared library.java.grpc_core // BEAM-11761
   compile library.java.grpc_context
+  permitUnusedDeclared library.java.grpc_context // BEAM-11761
   compile library.java.grpc_grpclb
+  permitUnusedDeclared library.java.grpc_grpclb // BEAM-11761
   compile library.java.grpc_netty
   compile library.java.grpc_netty_shaded
+  permitUnusedDeclared library.java.grpc_netty_shaded // BEAM-11761
   compile library.java.grpc_stub
+  permitUnusedDeclared library.java.grpc_stub // BEAM-11761
   compile library.java.grpc_google_cloud_pubsub_v1
   compile library.java.grpc_google_cloud_pubsublite_v1
+  permitUnusedDeclared library.java.grpc_google_cloud_pubsublite_v1 // BEAM-11761
   compile library.java.guava
+  compile library.java.hamcrest
+  compile library.java.http_client
   compile library.java.hamcrest_core
+  permitUnusedDeclared library.java.hamcrest_core // BEAM-11761
+  compile library.java.http_core
+  compile library.java.jackson_core
   compile library.java.jackson_databind
   compile library.java.joda_time
   compile library.java.junit
   compile library.java.netty_handler
   compile library.java.netty_tcnative_boringssl_static
+  permitUnusedDeclared library.java.netty_tcnative_boringssl_static // BEAM-11761
+  compile library.java.proto_google_cloud_bigquery_storage_v1
+  compile library.java.proto_google_cloud_bigquery_storage_v1beta1
```

Comment:
`proto_google_cloud_bigquery_storage_v1beta1` is introduced and marked as permitUnusedDeclared in this PR. It looks like it was removed recently, so please remove it: https://github.com/apache/beam/pull/13765/files#diff-098a130c2e6da0538aa36c54133e8cd0b44d171886934e3dd94e48d77a84a6d1R81.

Reply:
Ah yes. Should have noticed that after I merged with master.